woocommerce/woocommerce-ios
# [My Store Updates] Update top performers limit from 3 to 5

Closes: #5741 


### Description


As part of the My Store tab updates in the latest design (p91TBi-6Of-p2), we want to increase the maximum number of top performers from 3 to 5. I first thought we need to also update the Core Data `NSFetchedRequest`'s `fetchLimit`, but it turns out the top performer items are wrapped in one single Core Data `TopEarnerStats` object that is updated per time range on every API request. The final changes are:

- Added a `quantity` parameter to `StatsActionV4.retrieveTopEarnerStats` and updated `StatsStoreV4` accordingly with a test case
- Passed 3/5 to `quantity` parameter in `StoreStatsAndTopPerformersViewController`/`OldStoreStatsAndTopPerformersViewController` respectively

### Testing instructions


Prerequisite: the account is connected to a store with at least 5 products sold this year

- Launch the app
- After stats are loaded, check on the top performers section for each time range --> it should show max 5 top performers

- [x] @jaclync tests that the old stats show max 3 top performers when the feature flag is off

### Screenshots


feature flag on | feature flag off
-- | --
![Simulator Screen Shot - iPhone 11 - 2021-12-28 at 13 43 59](https://user-images.githubusercontent.com/1945542/147539324-65e1ff0a-ee48-4cd8-9b2e-339aca057924.png) | ![Simulator Screen Shot - iPhone 11 - 2021-12-28 at 13 44 43](https://user-images.githubusercontent.com/1945542/147539328-6c276c32-aac4-4d86-81ac-9a1e8a82afd8.png)


---
